arm/Microsoft.Compute/virtualMachines/readme.md

```diff
@@ -65,7 +66,7 @@ This module deploys one Virtual Machine with one or multiple nics and optionally
 | `encryptionAtHost` | bool | `True` |  | This property can be used by user in the request to enable or disable the Host Encryption for the virtual machine. This will enable the encryption for all the disks including Resource/Temp disk at host itself. For security reasons, it is recommended to set encryptionAtHost to True. Restrictions: Cannot be enabled if Azure Disk Encryption (guest-VM encryption using bitlocker/DM-Crypt) is enabled on your VMs. |
 | `extensionAntiMalwareConfig` | object | `{object}` |  | The configuration for the [Anti Malware] extension. Must at least contain the ["enabled": true] property to be executed |
 | `extensionCustomScriptConfig` | object | `{object}` |  | The configuration for the [Custom Script] extension. Must at least contain the ["enabled": true] property to be executed |
-| `extensionCustomScriptProtectedSetting` | object | `{object}` |  | Optional. Any object that contains the extension specific protected settings. |
+| `extensionCustomScriptProtectedSetting` | secureObject | `{object}` |  | Any object that contains the extension specific protected settings |
```

Comment:
is secureObject something already generated automatically?

Reply:
I was wondering the same thing and in fact, yes. 
![image](https://user-images.githubusercontent.com/5365358/163223090-5a3977e2-7896-405c-ac66-925672fd228b.png)